lib/crowbar/client/command/batch/build.rb

```diff
@@ -14,12 +14,30 @@
 # limitations under the License.
 #
 
+require "easy_diff"
+
 module Crowbar
   module Client
     module Command
       module Batch
         class Build < Base
           def request
+            args.easy_merge!(
+              includes: options.includes,
+              excludes: options.excludes
+            )
+
+            case args.file
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.